.travis.yml

```diff
@@ -23,4 +23,4 @@ before_script:
 
 script:
   - (cd $PHPCS_DIR ; ../../bin/phpunit --filter MO4)
-  - $PHPCS_DIR/bin/phpcs $MO4CS_DIR/Sniffs --standard=$PHPCS_DIR/phpcs.xml.dist --report=summary -np
+  - $PHPCS_DIR/bin/phpcs $MO4CS_DIR --standard=$PHPCS_DIR/phpcs.xml.dist --report=summary -np
```

Comment:
using `--report=diff` would help potential contrubutors here

Reply:
I think `--report=full` (which is default) is more valuable, since the violation is explained

Reply:
also 🆗 :+1: